hddm/models/hddm_stimcoding_nn.py

```diff
@@ -0,0 +1,193 @@
+from copy import copy
+import numpy as np
+from collections import OrderedDict
+
+from kabuki.hierarchical import Knode
+from kabuki.utils import stochastic_from_dist
+from hddm.models import HDDM
+from hddm.keras_models import load_mlp
+from hddm.cnn.wrapper import load_cnn
+import hddm
+
+import wfpt
+from functools import partial
+
+class HDDMnnStimCoding(HDDM):
```

Comment:
Also better to inherit here rather than copy.

Reply:
done